js/src/Builder.js

```diff
@@ -358,27 +369,18 @@ function load_map (map_data, should_update_data) {
   var s = this.selection
     .append('div').attr('class', 'search-menu-container')
     .append('div').attr('class', 'search-menu-container-inline')
-  var menu_div = s.append('div')
-  var search_bar_div = s.append('div')
-  var button_div = this.selection.append('div')
-
-  // Set up the search bar
-  this.search_bar = new SearchBar(search_bar_div, this.map.search_index,
-                                  this.map)
-  // Set up the hide callbacks
-  this.search_bar.callback_manager.set('show', function() {
-    this.settings_bar.toggle(false)
-  }.bind(this))
+  this.menu_div = s.append('div')
+  this.search_bar_div = s.append('div')
+  this.button_div = this.selection.append('div')
 
   // Set up the settings
   var settings_div = this.selection.append('div')
   var settings_cb = function (type, on_off) {
     // Temporarily set the abs type, for previewing it in the Settings menu
     var o = this.options[type + '_styles']
-    if (on_off && o.indexOf('abs') === -1) {
+    if (o !== null && o !== undefined && on_off && o.indexOf('abs') === -1) {
```

Comment:
This conditions look like they could be simplified:
```
if (o !== null && o !== undefined) {
  if (on_off && o.indexOf('abs') === -1) {
    o.push('abs')
  } else if (!on_off) {
    // ...
  }
}
```

Reply:
This code is no longer used